ts/de_DE.ts

```diff
@@ -52,7 +52,7 @@
     <message>
         <location filename="controller.py" line="1837"/>
         <source>Exporting G-code to</source>
-        <translation>Exportiere GCode nach</translation>
+        <translation>Exportiere gcode nach</translation>
```

Comment:
I changed it to gcode to have everywhere the same, there was gcode, GCode and G-Code. But agree to use G-Code that makes most sense.